lib/admin.js

```diff
@@ -37,7 +42,7 @@ const applyWriteConcern = require('./utils').applyWriteConcern;
  * @class
  * @return {Admin} a collection instance.
  */
-var Admin = function(db, topology, promiseLibrary) {
+const Admin = function(db, topology, promiseLibrary) {
```

Comment:
I'd go with `function Admin`.

Reply:
In that case, should I also change Db, Collection, and Cursor? I've kept them as `const Db = function`